Add all missing faculties to Freie Universitaet Berlin

jaczuu5456456345645x opened the pull request:
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

0. Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv"). **IF YOU DO NOT DO THIS, YOUR COMMIT WILL BE SUMMARILY REJECTED.**

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

2. _Insert new faculty **in alphabetical order**, not at the end of `csrankings-[a-z].csv`._ **Do not modify `csrankings.csv`, which is auto-generated.** **DO NOT MODIFY `csrankings-[0-9].csv`, WHICH ARE BEING DEPRECATED. USE `csrankings-[a-z].csv` INSTEAD.**

3. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [X] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [X] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [X] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**; please **do not** use the numeric suffixes, which are being obsoleted); include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [X] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [X] If the institution you are adding is not in the US,
update `country-info.csv`.

**(Advanced) Quick contribution via a shallow clone** 

A full clone of the CSrankings repository is almost 2GB.  To contribute a change without creating a full local clone of the CSrankings repo, you can do a _shallow clone_.  To do so, follow these steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)
2. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.
3. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to do another contribution and some time has passed, perform steps 1-3 again, creating a fresh fork and shallow clone.

Commit:
Adrian Paschke

Adrian Paschke, Freie Universitaet Berlin
no dblp-alias
https://dblp.org/pid/24/2942

Commit:
Daniel Göhring

Daniel Göhring, Freie Universitaet Berlin
dblp-alias Daniel Goehring
https://dblp.org/pid/97/3289

Commit:
Frank Noé

Frank Noé, Freie Universitaet Berlin
no dblp-alias
https://dblp.org/pid/46/6007

Commit:
Tim Landgraf

Tim Landgraf, Freie Universitaet Berlin
no dblp-alias
https://dblp.org/pid/04/10008

ghost commented:
**Adrian Paschke, Freie Universitaet Berlin**
no dblp-alias
https://dblp.org/pid/24/2942

**Agnès Voisard, Freie Universitaet Berlin**
no dblp-alias
https://dblp.org/pid/v/AVoisard

**Claudia Müller-Birn, Freie Universitaet Berlin**
dblp-alias Claudia Müller 0001
https://dblp.org/pid/05/1222

**Daniel Göhring, Freie Universitaet Berlin**
dblp-alias Daniel Goehring
https://dblp.org/pid/97/3289

**Frank Noé, Freie Universitaet Berlin**
no dblp-alias
https://dblp.org/pid/46/6007

**Gerhard Wunder, Freie Universitaet Berlin**
no dblp-alias
https://dblp.org/pid/63/5261

**Irene Ntoutsi, Freie Universitaet Berlin**
affiliation: University of Hannover -> Freie Universitaet Berlin
alias Eirini Ntoutsi (already with affiliation Freie Universitaet Berlin in csrankings-e.csv)
https://dblp.org/pid/n/IreneNtoutsi

**Knut Reinert, Freie Universitaet Berlin**
no dblp-alias
https://dblp.org/pid/10/884

**László Kozma 0002, Freie Universitaet Berlin**
no dblp-alias
https://dblp.org/pid/85/2288-2

**Lutz Prechelt, Freie Universitaet Berlin**
-> website update
no alias
https://dblp.org/pid/54/3155

**Marian Margraf, Freie Universitaet Berlin**
no dblp-alias
https://dblp.org/pid/14/4256

**Matthias Wählisch, Freie Universitaet Berlin**
-> website update
no alias
https://dblp.org/pid/91/5978

**Ralf Romeike, Freie Universitaet Berlin**
no dblp-alias
https://dblp.org/pid/02/4919

**Tim Landgraf, Freie Universitaet Berlin**
no dblp-alias
https://dblp.org/pid/04/10008

**Volker Roth 0002, Freie Universitaet Berlin**
no dblp-alias
https://dblp.org/pid/23/1185-2

Commit:
Knut Reinert

Knut Reinert,Freie Universitaet Berlin
no dblp-alias
https://dblp.org/pid/10/884

Commit:
László Kozma

László Kozma 0002, Freie Universitaet Berlin
no dblp-alias
https://dblp.org/pid/85/2288-2

Commit:
Gerhard Wunder

Gerhard Wunder, Freie Universitaet Berlin
no dblp-alias
https://dblp.org/pid/63/5261

Commit:
László Kozma

László Kozma, Freie Universitaet Berlin
no dblp-alias
https://dblp.org/pid/85/2288-2

Commit:
Merge branch 'emeryberger:gh-pages' into gh-pages

Commit:
Volker Roth

Volker Roth 0002, Freie Universitaet Berlin
no dblp-alias
https://dblp.org/pid/23/1185-2

Commit:
Agnès Voisard

Agnès Voisard, Freie Universitaet Berlin
no dblp-alias
https://dblp.org/pid/v/AVoisard

Commit:
Claudia Müller-Birn

Claudia Müller-Birn, Freie Universitaet Berlin
dblp-alias Claudia Müller 0001
https://dblp.org/pid/05/1222

Commit:
Marian Margraf

Marian Margraf, Freie Universitaet Berlin
no dblp-alias
https://dblp.org/pid/14/4256

Commit:
Merge branch 'emeryberger:gh-pages' into gh-pages

Commit:
Ralf Romeike

Ralf Romeike, Freie Universitaet Berlin
no dblp-alias
https://dblp.org/pid/02/4919

Commit:
Merge branch 'emeryberger:gh-pages' into gh-pages

Commit:
László Kozma

László Kozma, Freie Universitaet Berlin
no dblp-alias
https://dblp.org/pid/85/2288-2

Commit:
Merge branch 'emeryberger:gh-pages' into gh-pages

Commit:
Irene Ntoutsi, Freie Universitaet Berlin

alias Eirini Ntoutsi (already with affiliation Freie Universitaet Berlin in csrankings-e.csv)
https://dblp.org/pid/n/IreneNtoutsi

Commit:
Lutz Prechelt

update website
no alias
https://dblp.org/pid/54/3155

Commit:
Matthias Wählisch

update website
no alias
https://dblp.org/pid/91/5978

Commit:
Heiko Schwarz, Freie Universitaet Berlin

no alias
https://dblp.org/pid/91/3474

emeryberger commented on csrankings-h.csv:
```diff
@@ -442,6 +442,7 @@ Heiko Mantel,TU Darmstadt,http://www.mais.informatik.tu-darmstadt.de/Heiko_Mante
 Heiko Neumann,University of Ulm,http://www.informatik.uni-ulm.de/ni/mitarbeiter/HNeumann/index.html,0NbVSbMAAAAJ
 Heiko Paulheim,University of Mannheim,https://www.uni-mannheim.de/dws/people/professors/prof-dr-heiko-paulheim,SkSl3NkAAAAJ
 Heiko Röglin,University of Bonn,http://www.roeglin.org,NLLMDCkAAAAJ
+Heiko Schwarz,Freie Universitaet Berlin,https://iphome.hhi.de/schwarz/,41YpebQAAAAJ
```
Does not appear to be in CS. Please clarify.

jaczuu5456456345645x replied:
In October 2017, Heiko Schwarz became Professor at the FU Berlin. But it seems that he is also heading the research group "Image and Video Coding" at the Fraunhofer Heinrich Hertz Institute. Since I have not found anything official, that he appoints 75%+ of his time at FU Berlin, I removed him.

https://www.fu-berlin.de/informationen-fuer/beschaeftigte/personalia/neuberufen/neuberufene-aktuell/schwarz/index.html

Commit:
(Removed) Heiko Schwarz,Freie Universitaet Berlin

In October 2017, he became Professor at the FU Berlin. But it seems that he is also heading the research group "Image and Video Coding" at the Fraunhofer Heinrich Hertz Institute. Since I have not found anything official, that he appoints 75%+ of his at FU Berlin, I removed him.

Commit:
Merge branch 'emeryberger:gh-pages' into gh-pages
